content/tools/related-tools.md

```diff
@@ -60,6 +60,9 @@ People have written tools on top of Cucumber. These are the known third-party to
 | [OCCukes](https://github.com/royratcliffe/OCCukes)                                                              | Objective-C wire protocol implementation. Can be used for testing native iOS and Mac apps.                                                                                                                                                                                                   |
 | [Pickle](https://github.com/ianwhite/pickle/tree/master)                                                        | Easy model creation/reference in Cucumber - optionally leveraging your factories/blueprints                                                                                                                                                                                                  |
 | [Pickler](https://github.com/tpope/pickler/tree/master)                                                         | Synchronize user stories in [Pivotal Tracker](http://www.pivotaltracker.com/) with Cucumber Features.                                                                                                                                                                                        |
+| [Pitest-Cucumber-Plugin](https://github.com/alexvictoor/pitest-cucumber-plugin)                                 | Cucumber intergration for mutation testing tool PIT
+
+  |                                                                                     
```

Comment:
Something about this whitespace made the following lines weird. Try removing newlines "inside" the table cell.

Reply:
@olleolleolle , noticed it as well and fixed it